src/openMVG/types.hpp

```diff
@@ -28,19 +28,15 @@ typedef std::pair<IndexT,IndexT> Pair;
 typedef std::set<Pair> Pair_Set;
 typedef std::vector<Pair> Pair_Vec;
 
-#define OPENMVG_NO_UNORDERED_MAP 1
-
-#if defined OPENMVG_NO_UNORDERED_MAP
+#if defined OPENMVG_UNORDERED_MAP
```

Comment:
Ideally it would be created from cmake and configured.
I mean types.hpp normally must be a types.hpp.in (configured on the fly by cmake)